### DISCUSSION TITLE
Frontend API is Slow

### DISCUSSION THREAD

**Discussion by leonkosterss** [swellstores/community#238] [Help] [answered]
Hello,

We use the Frontend API to get the Products, based on a Category.

Most of the time, the response is very slow. Like 600ms - 1.85 sec.

See this page for an example: http://acc.obimex.nl/producten/plafondpanelen/?page=1

Sometimes requests are even 3 sec, for example this one:
```
https://obimex-bv.swell.store/api/products?sort=&expand%5B0%5D=categories&expand%5B1%5D=variants&expand%5B2%5D=attributes&categories%5B0%5D=63ce579da87f5100125d1580&categories%5B1%5D=64391fe12696db0012f65fb3&categories%5B2%5D=63cea43c7f89ce00126b4414&categories%5B3%5D=63ce57be2781a500127bb932
```

For us this is a real problem. Are there options to make this faster?

<img width="1093" alt="image" src="https://user-images.githubusercontent.com/51200981/235935275-3721f2f9-1a9f-49eb-9eec-d6d775f30693.png">

**Comment by logeshswell** [comment, marked as answer]
@leonkosterss Yes, in Community plan, we throttle API calls with additional delays and that causes delayed responses.
It is highly recommended to upgrade to Standard plan to have higher limits.
For more details: https://www.swell.is/pricing
<img width="1164" alt="Screenshot 2023-05-03 at 7 32 08 PM" src="https://user-images.githubusercontent.com/100766131/235938908-5883e37c-7dac-4834-8a29-c99d44fc2921.png">

**Comment by leonkosterss** [comment]
Aaaah nice! Can someone make us (algemeen@code14.nl) a Swell Partner? Then we can add our client with a Standard plan for now.

**Reply by logeshswell** [reply]
@StevenSwell Can you take this up?

**Reply by steven-k1** [reply]
@leonkosterss You can reach out to our partner team here: https://www.swell.is/become-a-partner#partner-form
